sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/clienttelemetry/ClientTelemetry.java

```diff
@@ -214,7 +214,10 @@ public boolean isClientMetricsEnabled() {
 
     public void init() {
         loadAzureVmMetaData();
-        sendClientTelemetry().subscribe();
+
+        if (this.isClientTelemetryEnabled()) {
+            sendClientTelemetry().subscribe();
```

Comment:
Should we add a changelog for this change?

Reply:
updated